src/Hangfire.Core/Dashboard/Content/js/Page.js

```diff
@@ -0,0 +1,490 @@
+
+                        if (!res1 && filterString != '') {
+                            parameters[parameters.length] = ["filterString", filterString].join('=');
+                            result = true;
+                        }
+
+                        if (!res2 && filterMethodString != '') {
+                            parameters[parameters.length] = ["filterMethodString", filterMethodString].join('=');  
+                            result = true;
+                        }
+                        
+                        if ( parameters.length > 0 ) url = '?' + parameters.join('&');
+                    }
+                    return result;
+                }
+
+                var updateOrRemoveSingleParameter = function (parameter, value) {
+                    var result = false;
+                    var parameters = url.substr(1).split('&');
+                    var elements;
+                    var i = 0;
+                    for (i = 0; i < parameters.length; i++) {
+                        elements = parameters[i].split('=');
+                        if (elements[0] == parameter && (typeof value == 'undefined' || value == '') ) {
+                            parameters.splice(i, 1);
+                            result = false;
+                            break;
+                        } else if (elements[0] == parameter && value != '') {
+                            elements[1] = value;
+                            parameters[i] = elements.join('=');
+                            result = true;
+                            break;
+                        } 
+                    }
+                    if (parameters.length > 0) url = '?' + parameters.join('&');
+                    return result;
+                }
+
+                var prepFilterDateTimeParameters = function (startDateTime, endDateTime) {
+                    if (typeof startDateTime == 'undefined' || typeof endDateTime == 'undefined') return false;
+                    var sDate = startDateTime.split(' ')[0].split('/'),
+                    eDate = endDateTime.split(' ')[0].split('/'),
+                    sTime = startDateTime.split(' ')[1].split(':'),
+                    eTime = endDateTime.split(' ')[1].split(':');
+
+                    
+                    var startSeconds = new Date(sDate[2],sDate[1],sDate[0], sTime[0],sTime[1]);
+                    var endSeconds = new Date(eDate[2], eDate[1], eDate[0], eTime[0],eTime[1]);
+                    var checked = $("#filterOnDateTime").is(':checked');
+                    if ((startSeconds - endSeconds) <= 0 && checked) {
+                        return addOrModifyDateTimeParameters(sDate, eDate, sTime, eTime);
+                    }
+                    else {
+                        return removeDateTimeParameters();
+                    }
+                }
+
+                var addOrModifyDateTimeParameters = function (sDate, eDate, sTime, eTime) {
```

Comment:
A lot of == usage in addOrModifyDateTimeParameters  that im not sure should be === ?